webui/src/pages/identity/GetUserStatistic.graphql

```diff
@@ -0,0 +1,13 @@
+query GetUserStatistic($authorQuery: String!, $participantQuery: String!, $actionQuery: String!) {
```

Comment:
You could pass just the ID and use the variable multiple time, no?

Reply:
Unfortunatlly not. I tried, but I cannot pass the ID variable to the query string in GraphQL.
I would like to have something like this.

```GraphQL
# Define the author query by concatenating the `author:` with the ID.
authored: allBugs(query: `author:$id`) {
   totalCount
},
participated: allBugs(query: `participant:$id`) {
   totalCount
},
```
Unfortunatly this doesn't work, as GraphQL does not support string concatenation. At least, that was the result of my research. That's why I have to construct the query in Typescript and pass the query as variable to graphql.
Not beautiful, but it works.

Reply:
hu, weird limitation indeed. Works for me!